libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -739,19 +761,25 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
                 st = State.keep;
             } else if (st == State.pathArgumentToAbsolute) {
                 st = State.keep;
-                auto p = buildNormalizedPath(workdir, arg).absolutePath;
-                rval.put(p);
-                includes.put(p);
+                putNormalizedAbsolute(arg);
             } else if (st == State.priorityKeepNextArg) {
                 st = State.keep;
                 rval.put(arg);
+	    } else if (st == State.checkingForQuotation) {
+		path.put(" " ~ arg);
+		if (isEndingWithQuotation(arg)) {
```

Comment:
I think `path.clear` has to be called after it is used because it may be reused for another "string building".